dev-packages/e2e-tests/test-applications/vue-3/tests/performance.test.ts

```diff
@@ -52,23 +50,18 @@ test('sends a navigation transaction with a parameterized URL', async ({ page })
     contexts: {
       trace: {
         data: {
-          params: {
-            id: '123',
-          },
           'sentry.source': 'route',
           'sentry.origin': 'auto.navigation.vue',
           'sentry.op': 'navigation',
+          'params.id': '456',
         },
         op: 'navigation',
         origin: 'auto.navigation.vue',
       },
     },
     transaction: '/users/:id',
     transaction_info: {
-      // So this is weird. The source is set to custom although the route doesn't have a name.
-      // This also only happens during a navigation. A pageload will set the source as 'route'.
-      // TODO: Figure out what's going on here.
-      source: 'custom',
+      source: 'route',
```

Comment:
@Lms24 I think I fixed the "issue" from this comment here 😅  (I check this both in the "old" format and in the new one, seems to be working in both now, hopefully.